exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,19 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+if (mark >= 80) {
+    return "A";
+  }
+  if (mark < 80 && mark > 60) {
```

Comment:
This is technically correct. However, just so you know, you could also write this using `else if`.